service/camera-streamer-pi219-8MP.service

```diff
@@ -17,8 +22,6 @@ ExecStart=/usr/local/bin/camera-streamer \
   -camera-low_res_factor=4 \
   ; bump brightness slightly
   -camera-options=brightness=0.1 \
```

Comment:
The ` \` to be removed from the end.

Reply:
Ah I missed that, thanks! Only got away with it because the following line was blank.